tests/links_tests/model_tests/vgg_tests/test_vgg16.py

```diff
@@ -5,17 +5,17 @@
 import chainer
 from chainer.initializers import Zero
 from chainer import testing
-from chainer.testing import attr
 from chainer import Variable
 
 from chainercv.links import VGG16
+from chainercv.testing import attr
 
 
 @testing.parameterize(
     {'pick': 'prob', 'shapes': (1, 200), 'n_class': 200},
     {'pick': 'pool5', 'shapes': (1, 512, 7, 7), 'n_class': None},
     {'pick': ['conv5_3', 'conv4_2'],
-     'shapes': ((1, 512, 14, 14), (1, 512, 28, 28)), 'n_class': None},
+     'shapes': ((1, 512, 14, 14), (1, 512, 28, 28)), 'n_class': None}
```

Comment:
What is the intent of this modification?

Reply:
No specific intention.  I can put it back